src/cloud-api-adaptor/entrypoint.sh

```diff
@@ -94,11 +94,11 @@ azure() {
 gcp() {
     test_vars GCP_CREDENTIALS GCP_PROJECT_ID GCP_ZONE PODVM_IMAGE_NAME
 
-    [[ "${PODVM_IMAGE_NAME}" ]] && optionals+="-gcp-image-name ${PODVM_IMAGE_NAME} "
+    [[ "${PODVM_IMAGE_NAME}" ]] && optionals+="-imageid ${PODVM_IMAGE_NAME} "
```

Comment:
Personally I think it's more intuitive to keep the options aligned to the cloud provider nuances to the extent possible. Like instance-type (aws), instance-size( azure).  
I see all public GCP docs refers to machine type (for cloud instances) so may be it's ok to use `machine-type` for GCP. Wdyt ?